caddy/README.md

```diff
@@ -29,55 +29,112 @@ Here's the things we find most useful:
 caddy file-server --browse --listen :4040
 ```
 
+### How to serve HTTPS on localhost
+
+Caddy can be used to test with https on localhost.
+
+`Caddyfile`:
+
+```Caddyfile
+https://localhost:443 {
+    handle /api/* {
+        reverse_proxy localhost:3000
+    }
+    
+    handle /* {
```

Comment:
```suggestion
    handle {
```

Reply:
Too much mental load to have multiple ways to do the same thing.

There should be 1 obvious way to do it.